schemas/definitions.json

```diff
@@ -315,28 +327,30 @@
       "attribution": {
         "type": "string",
         "title": "Attribution",
-        "description": "Short citation of the producer and publication year (and sometimes the dataset) that overrides the automatic attribution format `producer (year)` (where the year is automatically extracted from `date_published`).",
+        "description": "Citation of the data product to be used when the automatic format `producer (year)` needs to be overridden.",
         "requirement_level": "optional",
         "examples": [
-          "Statistical Review of World Energy, Energy Institute (2023)"
+          "Energy Institute - Statistical Review of World Energy (2023)"
         ],
         "examples_bad": [
-          ["Energy Institute - Statistical Review of World Energy (2023)"]
+          ["Statistical Review of World Energy, Energy Institute (2023)",
+          "Statistical Review of World Energy (Energy Institute, 2023)"]
         ],
         "guidelines": [
           ["Must start with a capital letter.", {"type": "exceptions", "value": ["The name of the institution or the author must be spelled with small letter, e.g. `van Haasteren`."]}],
           ["Must not end with a period."],
           ["Must end with the year of `date_published` in parenthesis."],
           ["Must not include any semicolon `;`."],
-          ["Should only be used if the automatic attribution format `producer (year)`is incomplete. For example, when the dataset title is well known and should be cited along with the producer, or when the original dataset version should also be mentioned."],
-          ["Should be constructed using placeholders, e.g. `{title}, {producer} ({year})`."]
+          ["Should only be used if the automatic attribution format `producer (year)` is considered uninformative. For example, when the title of the data product is well known and should be cited along with the producer, or when the original version of the data product should also be mentioned."],
+          ["Should follow the format `{producer} - {title} {version_producer} ({year})`."],
```

Comment:
I am a bit confused with this one. Thought `attribution` was free, but from this guideline it seems that there is a specific style we should follow?

Reply:
I've rephrased it. This is a "should" not a "must". My point here is that, if you are simply including the data product, then that should be the preferred way to do so. If you want to do something completely different, you are free.